guide/getting-started/visual-language.md

```diff
@@ -203,6 +223,35 @@ Think about Bitcoin’s visual language as a scale that ranges from muted to hig
 
 In digital product design, there are times when a minimally expressive approach is what’s needed. For example, when a user makes a transaction, we want them to focus. At other times, illustrations and stronger visual elements are more appropriate, and this is where we can take advantage of the creative freedom Bitcoin affords.
 
+## The Lightning network
+
+The Lightning network is a layer 2 solution for scaling Bitcoin. It allows people to make almost instant payments with very small fees.
+
+Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow icon of a lightning bolt. Sometimes, Lightning software and services may choose to completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.
```

Comment:
An attempt to simplify the paragraph. Should we mention the use of `emoji` I see a lot of applications using bolt emoji instead of the icons.

```suggestion
Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow lightning bolt icon. Sometimes, Lightning software and services completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.
```